MAUI/ComboBox/Selection.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: Selection in .NET MAUI ComboBox control | Syncfusion
+description: Learn here all about Selection modes support in Syncfusion .NET MAUI ComboBox (SfComboBox) control and more.
+platform: maui
+control: SfComboBox
+documentation: ug
+---
+
+# Selection in .NET MAUI ComboBox (SfComboBox)
+
+## Single selection
+
+The `ComboBox` allows user to select an single item from the drop-down list.
+
+### UI Selection 
+
+The selected item can be changed interactively by selecting from the drop-down list or entering the value.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfComboBox IsEditable="True"
+                    ItemsSource="{Binding SocialMedias}"
+                    DisplayMemberPath="Name"
+                    TextMemberPath="Name" />
+
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI ComboBox UI selection](Images/Selection/UISelection.png)
+
+
+### Programmatic selection 
+
+The selected item can be changed programmatically by using the `SelectedItem` or `SelectedIndex` property of `ComboBox` control. 
```

Comment:
The selected item can be changed programmatically by using the `SelectedItem` or `SelectedIndex` properties of the `ComboBox` control.